README.rst

```diff
@@ -57,9 +57,10 @@ Staging changes
 ---------------
 
 Buildbot changes can be tested on a staging machine.
-The docker registry will automatically build an image based on the staging branch, whenever it is updated.
+The Docker registry will automatically build an image based on the ``staging`` branch, whenever it is updated.
+
+Create a Fedora 20 instance on EC2 and note the IP of this instance.
 
-Create an Fedora 20 spot instance on EC2 and note the IP of this instance.
 In the following example the IP is 54.191.9.106.
 Set the Security Group of this instance to allow inbound traffic as shown below.
```

Comment:
This is unnecessary / should be changed now that there is a script. It is bad to maintain two methods of starting a Buildbot master (automated and manual).